packages/types/src/startSpanOptions.ts

```diff
@@ -20,6 +20,9 @@ export interface StartSpanOptions extends TransactionContext {
   /** An op for the span. This is a categorization for spans. */
   op?: string;
 
+  /** If set to true, this span will be forced to be treated as a transaction in the Sentry UI. */
+  forceTransaction?: boolean;
```

Comment:
m: I think we should be clear on the contract of this option. Meaning even if for now this will always apply we let users know that it'll not always have an effect. Cause eventually we'll deprecate the option but also ignore it if users opted into sending spans in segments (or whatever the payload/envelope type will be). 
 
```suggestion
  /** 
    * If set to true, this span will be forced to be treated as a transaction in the Sentry UI. 
    * 
    * This option only has an effect as long as the SDK is sending transactions. This might
    * change in the future via an opt-in mechanism, at which point and only if opted in,
    * the created span might no longer be sent as a transaction, despite this option
    * being set to `true`.
    */
  forceTransaction?: boolean;
```

Reply:
Maybe a bit more generic, something like:

```suggestion
  /** 
     * If set to true, this span will be forced to be treated as a transaction in the Sentry UI, if possible and applicable. 
     * Note that it is up to the SDK to decide how exactly the span will be sent, which may change in future SDK versions. 
     * It is not guaranteed that a span started with this flag set to `true` will be sent as a transaction.
     */
  forceTransaction?: boolean;
```

or something along these lines... not sure?

Reply:
Yup also fine with me!